ordered_model/models.py

```diff
@@ -80,8 +80,8 @@ def move(self, direction, qs=None):
             _("The method move() is deprecated and will be removed in the next release."),
             DeprecationWarning
         )
-        if direction == 'up':
-            self.up()
+        if direction in ['up', 'down', 'top', 'bottom']:
+            getattr(self, direction)()
         else:
```

Comment:
Looks like this branch isn't needed.

Also – the `move()` method is deprecated. Perhaps `move_view` should call `top()` and `bottom()` directly?